src/patternfly/layouts/Flex/flex.scss

```diff
@@ -21,6 +23,7 @@ $pf-l-flex--variable-map: build-variable-map("pf-l-flex--spacer", $pf-l-flex--sp
   display: var(--pf-l-flex--Display);
   flex-wrap: var(--pf-l-flex--FlexWrap);
   align-items: var(--pf-l-flex--AlignItems);
+  row-gap: var(--pf-l-flex--RowGap);
```

Comment:
I think this should probably be an opt in as it will add space between rows where there is none currently. I could see it being beneficial for sure, but likely to cause enough problems we should wait until a breaking change to make it the default. WDYT?